stable/phpmyadmin/README.md

```diff
@@ -82,7 +85,7 @@ $ helm install --name my-release -f values.yaml stable/phpmyadmin
 
 ## Upgrading
 
-### To 1.0.0
+### To 1.1.0
```

Comment:
Please do not bump the version here. The workaround shown below is to update from **0.x.x to 1.0.0**. Once you're on **1.0.0** you don't need to do it to upgrade to **1.1.0**